README.md

```diff
@@ -1902,6 +1902,7 @@ Update Time, five active automations, webhooks.
   * [OnlineExifViewer](https://onlineexifviewer.com/) — View EXIF data online instantly for a photo including GPS location and metadata.
   * [PDFMonkey](https://www.pdfmonkey.io/) — Manage PDF templates in a dashboard, call the API with dynamic data, and download your PDF. Offers 300 free documents per month.
   * [Pika Code Screenshots](https://pika.style/templates/code-image) — Create beautiful, customizable screenshots from code snippets and VSCode using the extension.
+  * [Postiz](https://postiz.com) — Self-hosted social media management software. It helps you to schedule posts, track the performance of your content, and manage all your social media accounts in one place.
```

Comment:
Does this entry tick all the boxes in the PR template?

Reply:
You are too fast :)
Updated